style2.css

```diff
@@ -1,261 +1,261 @@
-    font-family: Inter, sans-serif;
-    font-style: normal;
-    font-weight: 400;
-    font-size: 22px;
-    line-height: 43px;
-    display: flex;
-    align-items: center;
-    text-align: center;
-    color: white;
-    margin-left: 120px;
-    justify-content: space-between;
-    margin-bottom: auto;
-  }
-
-  .circle-container {
-    margin-left: 120px;
-    display: block ruby;
-  }
-
-  .rectangle-container {
-    margin-left: 115px;
-    display: block ruby;
-  }
-
-  /* stylelint-disable-next-line no-duplicate-selectors */
-}
+/* stylelint-disable selector-type-no-unknown */
+@media (min-width: 768px) {
+  html,
+  body {
+    width: 100vw;
+  }
+
+  #drop {
+    display: none;
+  }
+
+  .name {
+    margin-left: 154px;
+  }
+
+  .desktop-version {
+    display: ruby;
+    padding-right: 156px;
+    color: white;
+  }
+
+  .desktop-version img {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    margin-top: 55px;
+  }
+
+  .desktop-menu {
+    margin-right: 32px;
+  }
+
+  .upper-body-text h1 {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    font-family: "Crete Round", serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 72px;
+    line-height: 82px;
+    letter-spacing: 0.37px;
+    color: #ff6b00;
+    height: 123px;
+  }
+
+  .upper-body-text h2 {
+    color: #fff5e1;
+    font-family: Roboto, sans-serif;
+    font-size: 32px;
+    line-height: 44px;
+    height: 27px;
+  }
+
+  .upper-body-text p {
+    color: #fff;
+    font-family: Poppins, sans-serif;
+    font-size: 20px;
+    line-height: 32px;
+    height: 123px;
+    margin-right: 732px;
+  }
+
+  /* stylelint-disable-next-line no-duplicate-selectors */
+  .upper-body-text h1 {
+    margin-right: 732px;
+  }
+
+  .upper-body-text {
+    margin-left: 153px;
+  }
+
+  .upper-body {
+    background-image: url("upperimage.png");
+    background-repeat: no-repeat;
+    background-size: contain;
+  }
+
+  .myrecent {
+    display: flex;
+    margin-left: 153.5px;
+  }
+
+  .myrecent hr {
+    margin-top: 161px;
+    width: 794px;
+    border: 1px solid #212121;
+  }
+
+  .middle-image1 {
+    display: flex;
+  }
+
+  .myrecent h1 {
+    margin-top: 135px;
+    height: fit-content;
+    margin-left: 15px;
+  }
+
+  .middle-image {
+    display: none;
+  }
+
+  .middle-section {
+    flex-direction: column;
+    /* stylelint-disable-next-line declaration-block-no-redundant-longhand-properties */
+    flex-wrap: wrap;
+    display: flex;
+    /* stylelint-disable-next-line declaration-block-no-duplicate-properties */
+    flex-direction: row;
+    /* stylelint-disable-next-line declaration-block-no-duplicate-properties */
+    flex-wrap: wrap;
+    align-content: center;
+    margin-left: 141.5px;
+    margin-right: 118px;
+    margin-bottom: 130px;
+  }
+
+  .middle-image1-container {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-gap: 20px;
+    margin-left: 26px;
+  }

```

Comment:
- [ ] Please fix the work cards arrangement and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/160948292-6fc13b0d-a93d-4a8c-9df1-5aab1e9338db.png)  |![image](https://user-images.githubusercontent.com/68796232/160948418-c6ee61df-b4fe-410d-85d4-8296d3b1bb19.png)  |